CMakeLists.txt

```diff
@@ -33,7 +33,7 @@ set(CPACK_SOURCE_IGNORE_FILES "/build;/.git")
 include(CPack)
 
 # Add support for the "make uninstall" target
-configure_file("${CMAKE_MODULE_PATH}/cmake_uninstall.cmake.in"
+configure_file("${CMAKE_MODULE_PATH}/cmake/modules/cmake_uninstall.cmake.in"
```

Comment:
AFAIR I deleted usage of `CMAKE_MODULE_PATH`

Reply:
Not sure what I'm missing...

```
 35 # Add support for the "make uninstall" target
 36 configure_file("${CMAKE_MODULE_PATH}/cmake/modules/cmake_uninstall.cmake.in"
 37                "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake"
 38                IMMEDIATE @ONLY)
 39 add_custom_target(uninstall "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/cmake_uninst    all.cmake")
 40 

```

Reply:
https://github.com/NetBSD/pkgsrc-wip/blob/master/molsketch/patches/patch-CMakeLists.txt

Reply:
Embarrassing :(
Hopefully correct now...
Sorry!

Reply:
No worries, I have yet to get OpenBabel 3.0 (OpenSUSE doesn't seem to provide it yet ;) ). @krytarowski: You sure that's the way it should be? I haven't had much exposure to cmake...

Reply:
Yes.

Reply:
@hvennekate, we have a problem, then. Openbabel-3 is not in Fedora either, https://apps.fedoraproject.org/packages/openbabel

Reply:
> @hvennekate, we have a problem, then. Openbabel-3 is not in Fedora either, https://apps.fedoraproject.org/packages/openbabel

The Open Babel maintainer is aware and we are (slowly) working on it. There are several dependent packages that are quite dead upstream and that will probably break by the upgrade, so we need to prepare for the fallout. We may have something ready before the holidays (we are quite swamped at the moment) and in all likelihood updated packages will first appear in a copr. I will let you know.

Reply:
Hi Alex! Nice to see you again!
Yeah, I'll probably need until next weekend to get Openbabel-3 compiled and set up as well :( Sorry for the delay, @voidpin
What do you mean by "quite dead upstream"/fallout? Which packages would those be?

Reply:
The herd has really thinned out since I last checked, but still:

Link-time dependencies:
IQmol
avogadro
ghemical
gnome-chemistry-utils
kalzium
molsketch
xdrawchem

Run-time dependencies:
avogadro2
chemtool
molsketch

So far I've looked at chemtool, xdrawchem and gnome-chemistry-utils and none of them has received any love in quite some time. I still haven't checked which bits of Open Babel each program is using, that is my weekend project.

Reply:
Since I'm still working on remodeling the atom drawing (i.e. include the charge superscript in the bounding box), we might just merge it to the master branch here, but wait a bit until the actual release (on SourceForge) takes place?

Reply:
Nice hearing from you too Hendrik :smile:

Reply:
@hvennekate I'm fine by now :) molsketch is running fine with openbabel-3 on my NetBSD machine, thanks to @krytarowski.
I do hope you keep this project alive. `xdrawchem`, `gnome-chemistry-utils`, `chemtool` and `BKchem` seem to be dead. They haven't seen any updates in years.

Reply:
gnome-chemistry-utils needs either fixup in pkgsrc or removal.

I couldn't test it as it failed in an indirect dependency... IIRC in biology/chemical-mime-data.